packages/desktop-client/src/components/sidebar/SecondaryButtons.tsx

```diff
@@ -0,0 +1,38 @@
+// @ts-strict-ignore
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏new components we create ideally should be strict TS compliant.

```suggestion
```

Reply:
Removed